main.go

```diff
@@ -109,6 +115,25 @@ func xcodePath() (string, error) {
 	return split[0], nil
 }
 
+func xcodeVersion() float64 {
```

Comment:
Please use  `github.com/bitrise-io/go-xcode/utility.GetXcodeVersion` for retrieving Xcode version:

https://github.com/bitrise-io/go-xcode/blob/master/utility/utility.go#L44

Reply:
Done